Automation/General/qama and qawa.md

```diff
@@ -0,0 +1,35 @@
+> All code is guilty, until proven innocent.
+
+`qama` and `qawa` are our **template projects** for test automation.
+
+The idea behind them is to help you quickly start writing maintainable and easy to understand UI tests.
+While **qama** is designed to work primarily with mobile, **qawa** focuses on web projects.
+
+
+## Project structure
+
+Both projects are structured using the _page object model_.
+
+In short, that means that:
+
+- Locators are placed in a page class.
+- Tests are placed in a test class.
+
+Read [this page object model article](https://martinfowler.com/bliki/PageObject.html) for more information.
+
+
+## qama
+
+`qama` stands for `QA Mobile Automation.`
+It also happens to be a [sword](https://en.wikipedia.org/wiki/Qama). :)
+
+It is based on the [pytest framework](https://docs.pytest.org/en/7.1.x/index.html) and [Appium](https://appium.io). 
```

Comment:
Would it be enough to put `https://docs.pytest.org` only in the link? That way it will always point to the latest version when redirected (I guess).

Reply:
Of course. You are correct, it redirects to the latest one. Better to have it that way.
Updated.